### PR TITLE
Delete store, fix delete child category

### PR DESCRIPTION
**Pull request by mvn-vyle-dn**
## Description
- Delete Store
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: tests/Browser/Pages/Admin/Store/ListStores.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                ->assertSee('Store')
+                ->assertSee('Create Store')
+                ->assertSee('ID')
+                ->assertSee('Name')
+                ->assertSee('Address')
+                ->assertSee('Active')
+                ->assertSee('Show')
+                ->assertSee('Edit')
+                ->assertSee('Delete');
+    }
```
Comment (at-hienpham) [inline tests/Browser/Pages/Admin/Store/ListStores.php]: mấy cái assertSee() vỹ nên để trong dấu  __(' ')

File: tests/Browser/Pages/Admin/Store/ShowStores.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    {
+        $browser->assertPathIs($this->url())
+                ->assertSee('Detail Store')
+                ->assertSee('Name')
+                ->assertSee('Address')
+                ->assertSee('Phone')
+                ->assertSee('Uptime')
+                ->assertSee('Describe')
+                ->assertSee('Image')
+                ->assertSee('Edit')
```
Comment (at-hienpham) [inline tests/Browser/Pages/Admin/Store/ShowStores.php]: mấy cái assertSee() vỹ nên để trong dấu  __(' ')

File: tests/Browser/Pages/Admin/Store/ListStores.php (hunk 1 above)
Comment (at-vyle) [reply tests/Browser/Pages/Admin/Store/ListStores.php]: rứa là phải sửa lại hết mấy test kia hà :3

**Review by mvn-quangtran-dn — APPROVED** [review]
LGTM

**Pull request merged**